src/utils/tests/expressions.spec.js

```diff
@@ -29,10 +29,6 @@ describe("expressions", () => {
     it("sanitizes 2 quotes", () => {
       expect(sanitizeInput('"3"')).toEqual('"3"');
     });
-
-    it("sanitizes forward slashes", () => {
-      expect(sanitizeInput("foo\\\\")).toEqual("foo\\\\\\\\");
-    });
   });
 
   describe("getValue", () => {
```

Comment:
and are there others that we need? perhaps where we want to guard the back slash?

Reply:
I can't think of any cases where we would need it. If you have any I'm more than happy to add them.